jekyll/_cci2/config-policies-for-self-hosted-runner.adoc

```diff
@@ -0,0 +1,143 @@
+---
+description: Learn how to write a policy to restrict which projects can run jobs on a self-hosted runner.
+contentTags:
+  platform:
+  - Cloud
+---
+= Config policies for self-hosted runner (Open-preview)
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+NOTE: The config policies feature is available on the **Scale** plan and is currently in **open preview**.
+
+CAUTION: While the config policies feature is in **open preview**, a service failure will result in build configurations **not** being evaluated against policies. This should be taken into consideration before using config policies for compliance purposes.
+
+Follow this how-to guide to learn how to create a config policy to restrict which projects can run jobs on a self-hosted runner resource class. For more information about config policies, see the xref:config-policy-management-overview#[Config policies overview].
+
+[#prerequisites]
+== Prerequisites
+
+* Install/update the CircleCI CLI, and ensure you have authenticated with a token before attempting to use the CLI with config policies. See the link:/docs/local-cli[Installing the Local CLI] page for more information.
+
+* Set up a self-hosted runner. See the xref:runner-installation#[Installing self-hosted runners] guide.
+
+* Ensure you have **enabled** config policy evaluation for your organization so that project configurations **will** be evaluated against your organization's policies when pipelines are triggered.:
++
+[source,shell]
+----
+circleci policy settings --enabled=true --owner-id <your-organization-ID>
+----
++
+Example output:
++
+[source,shell]
+----
+{
+  "enabled": true
+}
+---- 
++
+{% include snippets/find-organization-id.adoc %}
+
+[#create-your-policy]
+== 1. Create your policy
+
+. If you have not already done so, create an empty directory to store your policies. For example:
++
+[source,shell]
+----
+mkdir ./config-policies
+----
+
+. Create a new file in this directory for your new policy. Name the file `runner.rego`
```

Comment:
There should be a mention towards managing policies via VCS, we don't want users to be creating policies from many sources as policies need one centralized source

Reply:
OK @davidmdm I've made a separate PR to move the management through VCS steps to be part of the initial setup and recommendations, rather than just being presented as an "option" https://github.com/circleci/circleci-docs/pull/7829/files. I'll handle this change separately to this policies for runner PR if that works